osu.Game/Overlays/BeatmapSet/BasicStats.cs

```diff
@@ -60,7 +60,7 @@ private void updateDisplay()
             }
             else
             {
-                length.Value = TimeSpan.FromSeconds(beatmap.OnlineInfo.Length).ToString(@"m\:ss");
+                length.Value = TimeSpan.FromSeconds(beatmap.Length).ToString(@"m\:ss");
```

Comment:
isn't this still wrong?

Reply:
oops, looks like i missed it while searching for them...